blathering-barnacles/Bangazon-2
# Search bar

Description: Added the ability to search for items in the db and display them in the "home" page.

Steps to Test:
* Click on the input box at the sidebar.
* Try different combination of characters, if any of those match with a product in the database it should be displayed with price, and quantity.

Link to Feature Ticket: [Ticket 4](https://github.com/blathering-barnacles/Bangazon-2/issues/4)
